dbt_copilot_helper/templates/addons/env/aurora-postgres.yml

```diff
@@ -151,3 +152,291 @@ Resources:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      Description: Ingress from Lambda Functions to DB
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 5432
+      ToPort: 5432
+      SourceSecurityGroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+
+  {{ service.prefix }}SecretsManagerIngress:
+    Metadata:
+      'aws:copilot:description': 'Allow ingress from Lambda Functions in my application to the Secrets Manager'
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      Description: Ingress from Lambda Functions to Secrets Manager
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 443
+      ToPort: 443
+      SourceSecurityGroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+
+  {{ service.prefix }}LambdaEgress:
+    Metadata:
+      'aws:copilot:description': 'Allow egress from DB cluster in my application to the Lambda Function'
+    Type: AWS::EC2::SecurityGroupEgress
+    Properties:
+      Description: Egress from DB to Lambda Functions
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 5432
+      ToPort: 5432
+      DestinationSecurityGroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+
+  {{ service.prefix }}SecretsManagerEgress:
+    Metadata:
+      'aws:copilot:description': 'Allow egress from Secrets Manager in my application to the Lambda Function'
+    Type: AWS::EC2::SecurityGroupEgress
+    Properties:
+      Description: Egress from Secrets Manager to Lambda Functions
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 443
+      ToPort: 443
+      DestinationSecurityGroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+
+  {{ service.prefix }}HTTPSEgress:
+    Metadata:
+      'aws:copilot:description': 'Allow egress for HTTPS (so the Lambda Function can post a success response back to the Custom Resource)'
+    Type: AWS::EC2::SecurityGroupEgress
+    Properties:
+      CidrIp: 0.0.0.0/0
+      Description: Egress for HTTPS
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 443
+      ToPort: 443
+
+  {{ service.prefix }}LambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Sub "${App}-${Env}-aurora-create-application-user"
+      Handler: index.handler
+      Runtime: python3.11
+      Layers:
+        - arn:aws:lambda:eu-west-2:763451185160:layer:python-postgres:1
+      MemorySize: 128
+      Timeout: 10
+      Role: !GetAtt {{ service.prefix }}LambdaFunctionExecutionRole.Arn
+      VpcConfig:
+        SecurityGroupIds:
+          - !Ref {{ service.prefix }}DBClusterSecurityGroup
+        SubnetIds: !Split [ ",", !Ref PrivateSubnets ]
+      Code:
+        ZipFile: |
```

Comment:
Any chance this can be extracted into a python file so it can be tested? Myself and @antroy-madetech are doing this for [creating a healthcheck file for the s3 proxy](https://github.com/uktrade/copilot-tools/compare/main...DBTP-457-s3-object-for-healthchecks) and we will read in the text content of the function when generating the template.

Reply:
In short, yes, but not in this pull request.

See the first bullet on [this comment](https://uktrade.atlassian.net/browse/DBTP-309?focusedCommentId=125191) from @yusufsheiqh.

Also, when you have yours working, that will probably inform how we get this out to a python file.